staging/vhost-device-gpu/src/virtio_gpu.rs

```diff
@@ -387,23 +383,27 @@ impl RutabagaVirtioGpu {
         })
     }
 
-    fn configure_rutabaga_builder(gpu_mode: GpuMode) -> RutabagaBuilder {
-        let component = match gpu_mode {
+    fn configure_rutabaga_builder(gpu_config: GpuConfig) -> RutabagaBuilder {
```

Comment:
clippy -- -W clippy::pedantic
```suggestion
    fn configure_rutabaga_builder(gpu_config: &GpuConfig) -> RutabagaBuilder {